ray_julia_jll/src/wrappers/any.jl

```diff
@@ -159,7 +159,11 @@ NullPtr(::Type{Buffer}) = BufferFromNull()
 
 FromHex(::Type{ObjectID}, str::AbstractString) = ObjectIDFromHex(str)
 FromRandom(::Type{ObjectID}) = ObjectIDFromRandom()
-Nil(::Type{ObjectID}) = ObjectIDFromNil()
+FromNil(::Type{ObjectID}) = ObjectIDFromNil()
```

Comment:
I was trying to match the C++ interface with calling this function `Nil`. i.e `ObjectID::FromHex` and `ObjectID::Nil` exist but `ObjectID::FromNil` does not.

Reply:
Yup, this was intentional: `Nil` returns a _reference_ and it screws up downstream operations with `==`, `hash`, etc.